cdap-cli/src/main/java/co/cask/cdap/cli/command/metadata/AddMetadataPropertiesCommand.java

```diff
@@ -34,6 +34,10 @@
 
   private final MetadataClient client;
 
+  public static final String ENTITY_DESCRIPTION_STRING = String.format("'<%s>' is of the form " +
```

Comment:
Doesn't seem like this should be defined here. Its used here as well as some security/authorization commands right? Perhaps define it in ``AbstractCommand`` or similar?